mobile/src/main/java/ch/epfl/sdp/mobile/ui/profile/Match.kt

```diff
@@ -0,0 +1,60 @@
+package ch.epfl.sdp.mobile.ui.profile
+
+import androidx.compose.material.ExperimentalMaterialApi
+import androidx.compose.material.Icon
+import androidx.compose.material.ListItem
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.vector.ImageVector
+import ch.epfl.sdp.mobile.ui.i18n.LocalizedStrings
+import ch.epfl.sdp.mobile.ui.social.Loss
+import ch.epfl.sdp.mobile.ui.social.MatchResult
+import ch.epfl.sdp.mobile.ui.social.Tie
+import ch.epfl.sdp.mobile.ui.social.Win
+
+/**
+ * Composes a Match log using a match [title], [subtitle] and an [icon]
+ * @param title match title
+ * @param subtitle match subtitle info
+ * @param icon match icon
+ * @param modifier the [Modifier] for this composable.
+ */
+@OptIn(ExperimentalMaterialApi::class)
+@Composable
+fun Match(
+    title: String,
+    subtitle: String,
+    icon: ImageVector,
+    modifier: Modifier = Modifier,
+) {
+  ListItem(
+      modifier = modifier,
+      icon = { Icon(icon, null) },
+      text = { Text(title) },
+      secondaryText = { Text(subtitle) },
+  )
+}
```

Comment:
Good idea to extract this in its own file. However, I don't see the point of this composable, given that the caller has to pass in the subtitle too. What would you think of refactoring it (alongside with `chooseSubtitle` to have a signature a bit more like this one ?

```kotlin
@Composable
fun Match(
  match: ChessMatch,
  modifier: Modifier = Modifier,
) {
  // ...
}
```

This way, the function `chooseSubtitle` could be kept `private` and the composable have an actual utility.